joomla/joomla-cms
# [4.0] Fix alerts transparent background

Pull Request for Issue # .

### Summary of Changes



### Testing Instructions
Apply the patch and check the backend when saving an article or saving the global configuration


### Expected result



### Actual result



### Documentation Changes Required
Nope
